theano/gpuarray/basic_ops.py

```diff
@@ -319,7 +319,9 @@ def c_header_dirs(self):
     def _generate_kernel_code(self, k):
         code = '\\n'.join(l for l in k.code.split('\n'))
         code = code.replace('"', '\\"')
-        return ("""static const char *%(cname)s = "%(code)s";""" %
+        return ("""static const char *%(cname)s_unsigned = "%(code)s";
+                static const char *%(cname)s = (char *)%(cname)s_unsigned;
```

Comment:
This fix a compilation error with newer g++ that are more strict and don't like that some of the values are more then 128. So this is still needed. the newer g++ is probably still not supported everywhere.